data/stories/get_started_playground.yml

```diff
@@ -0,0 +1,85 @@
+version: "2.0"
+stories:
+- story: Getting started on Playground Page
+  steps:
+  - intent: get_started_playground
+  - action: utter_ask_playground_help
+  - intent: affirm
+  - action: utter_what_help
```

Comment:
This is an NLU problem as well as dialogue- where do we handle specific problem reports?
```
Your input ->  /get_started_playground                      
Do you have any questions or issues you're running into with Rasa Playground?
Your input ->  yes                                          
Ok, what can I help you with?
Your input ->  it's broken                                  
Can you describe the problem you're having?
Your input ->  it's not training                            
Ok, what can I help you with?
Your input ->  the playground, it's not training my model   
? Sorry, I'm not sure I've understood you correctly 🤔 Do yo
u mean...  (Use arrow keys)                                 
 » 1: nlu_fallback (/nlu_fallback{"product": "playground"})
   2: Something else (/out_of_scope)
   Type out your own message...    
 ```